test/test_helper.rb

```diff
@@ -2,6 +2,7 @@
 require 'minitest/autorun'
 require 'fakeweb'
 require 'mocha/setup'
+require 'pry'
```

Comment:
✂️

Reply:
😢 I like pry though!

Is this really not needed? Without it, dropping `binding.pry` into code/tests fails with an undefined method error.

Reply:
agreed. Might make sense as its own PR mind you. Is there anyway we can autoload it in development so we don't need to require it and remove it? I often like to use it outside the test_helper

Reply:
ah, fair enough. didn't realize your intention was to keep the dependency in, I thought this was just leftover from your debugging. I'm fine to keep it 👍

Reply:
@jtgrenz I think if we loaded it in 'development', we'd be requiring it within client code, which we definitely don't want to do. If you're working with shopify_api in a project, that project can require 'pry' in whatever environments it wants.